src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/high-risk-data-breaches/HighRiskBreachContent.tsx

```diff
@@ -0,0 +1,164 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import CreditCardIllustration from "../images/high-risk-data-breach-credit-card.svg";
+import BankAccountIllustration from "../images/high-risk-data-breach-bank-account.svg";
+import pinIllustration from "../images/high-risk-data-breach-pin.svg";
+import SocialSecurityNumberIllustration from "../images/high-risk-data-breach-ssn.svg";
+import NoBreachesIllustration from "../images/high-risk-breaches-none.svg";
+import { useL10n } from "../../../../../../../hooks/l10n";
+
+export const HighRiskBreachContent = (typeOfBreach: string) => {
+  const l10n = useL10n();
+  let title,
+    secondaryDescription,
+    recommendationSteps,
+    breachIllustration,
+    illustrationAlt;
+  // TODO: Expose email list & count here https://mozilla-hub.atlassian.net/browse/MNTOR-2112
+  const emailsMonitored = ["email1@gmail.com", "email2@gmail.com"]; // mocked
+  const emailsFormatter = new Intl.ListFormat("en-US", {
+    style: "long",
+    type: "conjunction",
+  });
+
+  const CreditCardRecommendationSteps = (
+    <ol>
+      <li>{l10n.getString("high-risk-breach-credit-card-step-one")}</li>
+      <li>{l10n.getString("high-risk-breach-credit-card-step-two")}</li>
+      <li>{l10n.getString("high-risk-breach-credit-card-step-three")}</li>
+    </ol>
+  );
+
+  const BankAccountRecommendationSteps = (
+    <ol>
+      <li>{l10n.getString("high-risk-breach-bank-account-step-one")}</li>
+      <li>{l10n.getString("high-risk-breach-bank-account-step-two")}</li>
+      <li>{l10n.getString("high-risk-breach-bank-account-step-three")}</li>
+    </ol>
+  );
+
+  const stepOneSSNLink =
+    "https://consumer.ftc.gov/articles/what-know-about-credit-freezes-fraud-alerts";
+  const stepTwoSSNLink = "https://www.annualcreditreport.com/index.action";
+
+  const SocialSecurityNumberRecommendationSteps = (
+    <ol>
+      {/* TOOD: Add question mark modal explaining the SSN breach resolution - MNTOR-2127 */}
+      <li>
+        {l10n.getFragment("high-risk-breach-social-security-step-one", {
+          elems: {
+            link_to_info: <a href={stepOneSSNLink} />,
```

Comment:
Do these need `target="_blank"` and `rel="noopener noreferrer"` since they're external links (here and L59 below)